pkgs/by-name/ku/kubectl-kcl/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-kcl";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = pname;
+    rev = "v${version}";
```

we should use `tag` to get a tagged release
```suggestion
    tag = "v${version}";
```

---

amended